EIPS/eip-5450.md

```diff
@@ -87,8 +87,8 @@ For each instruction:
       - after any other non-terminating instruction stack height bounds are adjusted by subtracting the number of instruction inputs and adding the number of instruction outputs,
       - terminating instructions do not need to update stack height bounds.
 3. Determine the list of successor instructions that can follow the current instructions:
-   1. The next instruction for all instructions other than terminating instructions and unconditional jump.
-   2. All targets of a conditional or unconditional jump.
+   1. The next instruction if the current instruction is not a terminating instruction nor unconditional jump.
+   2. All the current instruction targets if the current instruction is a conditional or unconditional jump.
```

Comment:
I think for terms conditional and unconditional jumps we should list the instructions.

Reply:
done